hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -779,19 +812,19 @@ static u8 *xpadneo_report_fixup(struct hid_device *hdev, u8 *rdesc, unsigned int
 		}
 	}
 
-	/* fixup reported button count for Xbox controllers in Linux mode */
-	if (*rsize >= 164) {
-		/* 11 buttons instead of 10: properly remap the Xbox button */
-		if (rdesc[140] == 0x05 && rdesc[141] == 0x09 &&
-		    rdesc[144] == 0x29 && rdesc[145] == 0x0F &&
-		    rdesc[152] == 0x95 && rdesc[153] == 0x0F &&
-		    rdesc[162] == 0x95 && rdesc[163] == 0x01) {
-			hid_notice(hdev, "fixing up button mapping\n");
-			rdesc[145] = 0x0B;	/* 15 buttons -> 11 buttons */
-			rdesc[153] = 0x0B;	/* 15 bits -> 11 bits buttons */
-			rdesc[163] = 0x05;	/* 1 bit -> 5 bits constants */
-		}
-	}
+	// /* fixup reported button count for Xbox controllers in Linux mode */
```

Comment:
Will add back, just wanted to start from a clean slate

Reply:
It may be easier to add `#if 0` in front and `#endif` at the end of the block. Then you can turn it on and off by just swapping 0 with 1.

Reply:
I just use `Ctrl+/` and VS code takes care of the (un)commenting :)

Reply:
But the diff is bigger and sometimes harder to read... ;-) But it's fine - do what works for you.